convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -164,31 +179,59 @@ def run(self):
                         " environment variable.  Please switch to 'CONVERT2RHEL_ALLOW_OLDER_VERSION'"
                         " instead."
                     )
+                    self.add_message(
+                        level="WARNING",
+                        id="DEPRECATED_ENVIRONMENT_VARIABLE",
+                        title="Deprecated environment variable",
+                        description="A deprecated environment variable has been detected",
+                        diagnosis="You are using the deprecated 'CONVERT2RHEL_UNSUPPORTED_VERSION'",
+                        remediation="Please switch to the 'CONVERT2RHEL_ALLOW_OLDER_VERSION' environment variable instead",
+                    )
 
-                logger.warning(
+                diagnosis = (
                     "You are currently running %s and the latest version of Convert2RHEL is %s.\n"
                     "'CONVERT2RHEL_ALLOW_OLDER_VERSION' environment variable detected, continuing conversion"
                     % (installed_convert2rhel_version, latest_available_version[1])
                 )
-
+                logger.warning(diagnosis)
+                self.add_message(
+                    level="WARNING",
+                    id="ALLOW_OLDER_VERSION_ENVIRONMENT_VARIABLE",
+                    title="Outdated Convert2RHEL version detected",
+                    description="An outdated Convert2RHEL version has been detected",
+                    diagnosis=diagnosis,
```

Comment:
We should use convert2rhel, we only use Convert2RHEL in downstream
```suggestion
                    title="Outdated convert2rhel version detected",
                    description="An outdated convert2rhel version has been detected",
                    diagnosis=diagnosis,
```